core.d.ts

```diff
@@ -419,7 +419,10 @@ if (stream2.fileType?.mime === 'image/jpeg') {
 export function fileTypeStream(readableStream: ReadableStream, options?: StreamOptions): Promise<ReadableStreamWithFileType>;
 
 /**
-Detect the file type of a [`Blob`](https://nodejs.org/api/buffer.html#class-blob).
+Detect the file type of a [`Blob`](https://nodejs.org/api/buffer.html#class-blob) or .
+
+@param blob [`Blob`](https://nodejs.org/api/buffer.html#class-blob) used for file detection
```

Comment:
This is moot. You are restarting the type which is already made clear in the type signature, and the description doesn't add anything.

Reply:
I wanted explictly mention the fact that this is the way to parse [File](https://developer.mozilla.org/en-US/docs/Web/API/File), as users keep being surprised the Node.js specific function `fileTypeFromFile` cannot be used in a browser environment.